pyroomacoustics/room.py

```diff
@@ -592,6 +592,11 @@ def callback_mix(premix, snr=0, sir=0, ref_mic=0, n_src=None, n_tgt=None):
                 "RT60 between the {}th mic and {}th source: {:.3f} s".format(m, s, rt60[m, s])
             )
 
+Free-field simulation
+=====================
+
+You can also use this package to simulate free-field sound propagation between a set of sound sources and a set of microphones, without considering room effects. To this end, you can use the :py:obj:`pyroomacoustics.room.AnechoicRoom` class, which simply corresponds to setting the maximum image image order of the room simulation to zero. This allows for early development and testing of various audio-based algorithms, without worrying about room acoustics at first. Thanks to the modular framework of pyroomacoustics, room acoustics can easily be added, after this first testing stage, for more realistic simulations. 
+
```

Comment:
Can you please add a link to the relevant example in the paragraph ?

Also, a short code snippet would be awesome, but I leave that to you to decide! If you don't feel like it, don't do it 😄